metadata-service/factories/src/main/resources/application.yml

```diff
@@ -26,7 +26,7 @@ authentication:
 
 ingestion:
   enabled: ${UI_INGESTION_ENABLED:true}
-  defaultCliVersion: '${UI_INGESTION_DEFAULT_CLI_VERSION:0.8.26.3}'
+  defaultCliVersion: '${UI_INGESTION_DEFAULT_CLI_VERSION:0.8.26.6}'
```

Comment:
If we set default here why do we need to set docker env?

Reply:
I added it here because it was already here. Ideally we should just update this in docker env going forward.